test/spec/provider/bpmn/ElementNameSpec.js

```diff
@@ -370,6 +371,93 @@ describe('element-name-properties', function() {
 
     });
 
+    describe('of a group', function() {
```

Comment:
TODO: 

* Verify that `no name` -> `name is set` -> `CategoryValue` is created and properly linked to the document model.